includes/stats-calculator.php

```diff
@@ -233,6 +233,43 @@ public function get_attendees_not_contributing( int $event_id ): array {
 		return $attendees_not_contributing;
 	}
 
+	/**
+	 * Get the hosts' users for an event.
+	 *
+	 * @param int $event_id The id of the event.
+	 * @return array[WP_User] The hosts of the event.
+	 */
+	public function get_hosts( int $event_id ): array {
```

Comment:
I think we should place this method in the attendee repository, something like:

```php
/**
 * @return Attendee[] Attendees of the event who are hosts.
 */
public function get_hosts( int $event_id ): array;
```

There's already a `get_attendees()` method in the repository (though it's currently not implemented), so I think it would make sense to put the `get_hosts()` method there too.

Reply:
Done. Thank you for the feedback!